Schmidt/template/README.md

```diff
@@ -1,13 +1,13 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+
+factorial is a function that returns something important.
 
     >>> from simple_functions import factorial
     >>> factorial(10)
     9
```

Comment:
10! is not 9, this should be 3,628,800 instead.

Reply:
done!